src/odb/src/db/dbObstruction.cpp

```diff
@@ -93,6 +94,12 @@ dbIStream& operator>>(dbIStream& stream, _dbObstruction& obs)
   stream >> obs._min_spacing;
   stream >> obs._effective_width;
 
+  _dbDatabase* db = obs.getImpl()->getDatabase();
+  if (!db->isSchema(db_schema_except_pg_nets_obstruction)) {
+    // assume false for older databases
+    obs._flags._except_pg_nets = false;
+  }
```

Comment:
don't you need an else to actually read the field?

Reply:
the flags size didn't change, so I don't think thats needed (since it's still a 32 bit word it's reading (a few lines before this), but it's not clear if the unused bits are set of not, so this just makes sure for older versions it's set to false. I could be wrong, but I don't see how to implement this with the bit_fields thing.